styles/styles.css

```diff
@@ -0,0 +1,485 @@
+/* CSS Mini Reset */
+
+html,
+body,
+div,
+form,
+fieldset,
+legend,
+caption,
+label,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre
+{
+    font-size: 100%;
+    font-weight: normal;
+
+    margin: 0;
+    padding: 0;
+
+    vertical-align: baseline;
+
+    border: 0;
+    outline: 0;
+    background: transparent;
+}
+
+img
+{
+    border: 0;
+}
+
+body, html {
+    font-family: 'Open Sans', sans-serif;
+    font-weight: normal;
+    color: rgb(94, 94, 94);
+    height: 100%;
+}
+
+a {
+    color: rgb(94, 94, 94);
+    text-decoration: none;
+}
+
+a:hover {
+    color: rgb(189, 141, 149);
+}
+
+#header {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#page_heading {
+    text-transform: uppercase;
+    letter-spacing: 0.5rem;
+    font-size: 2.5rem;
+    margin-top: 4rem;
+    margin-bottom: 0;
+}
+
+#heading_subtext {
+    text-transform: uppercase;
+    letter-spacing: 0.05rem;
+    font-size: 0.8rem;
+    font-weight: lighter;
+    margin-top: 15px;
+}
+
+#divider_container {
+    
+    height: 10px;
+    background-color: white;
+    margin-top: 70px;
+}
+
+.divider {
+    border-top: solid rgb(184, 184, 184) 1px;
+    border-bottom: solid rgb(184, 184, 184) 1px;
+    height: 5px;
+    width: 70vw;
+}
+
+#features_caption {
+    margin-top: 70px;
+    text-transform: uppercase;
+    letter-spacing: 0.05rem;
+}
+
+#features_img_container {
+    display: flex;
+    justify-content: center;
+}
+
+.features_img {
+    max-width: 250px;
+    margin: 12px;
+    margin-top: 30px;
+}
+
+#features_button_container {
+    display: flex;
+    justify-content: center;
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+button {
+    height: 40px;
+    width: 180px;
+    border: none;
+    border-radius: 3px;
+    background-color: rgb(189, 141, 149);
+    color: white;
+    text-transform: uppercase;
+    font-size: 0.7rem;
+    box-shadow: 2px 2px rgb(218, 218, 218);
+    margin: 20px;
+}
+
+button:hover {
+    background-color: rgb(175, 128, 136);
+    cursor: pointer;
+}
+
+#banner_img_container {
+    position: relative;
+    text-align: center;
+    width: 100%;
+    margin-top: 40px;
+}
+
+
+#banner_img {
+    width: 100%;
+}
+
+#img_quote {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  font-size: 2rem;
+  text-transform: uppercase;
+  color: black;
+}
+
+
+/* All blog post content */
```

Comment:
I like that you've split up your css into sections like this one and the ones below as well. When doing one large CSS file for the whole project this is a good practice. You could even go as far as to create one section per page if there are enough unique ids for each page.